.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,46 @@
+---
+name: "🐛 Bug Report"
+about: Report a reproducible bug or regression.
+title: 'Bug: '
+labels: ['unconfirmed', 'bug']
+
+---
+
+# Prerequisites
+
+Please answer the following questions for yourself before submitting an issue.
```

Comment:
Change to "Please answer the following questions before submitting an issue"

Reply:
Fixed.